### DISCUSSION TITLE
World Record for Coldest Inflatable Hot Tub?

### DISCUSSION THREAD

**Discussion by rosetown-kelly** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#409] [General]
Do you think it's possible I've set the world record for a a Lay-z-Spa in the coldest temperature? 

-37 Celsius and somehow it's maintaining temperature. RIP my electricity bill, however.  

![image](https://user-images.githubusercontent.com/79489853/206097774-72323edb-82f6-4193-891d-6f0adfe09459.png)

**Comment by visualapproach** [comment]
Impressive!
I imagine getting out of the tub is like the terminator scene where he gets liquid nitrogen over him 😀

**Comment by chunkysteveo** [comment]
I don't want to know your energy bill for this??!! I hope you get good rates??!! What steps do you do to help the tub? Insulation, covered, etc?

**Comment by chunkysteveo** [comment]
In the UK, my electricity is currently about £0.33/kWh. 

So lets say I treat myself for Christmas and fill and turn up to 40°C today, as we are having a cold snap and it's about 0°C.... assuming the spoken average of the 2kW heating element is about 1°C per hour to heat (depending on your ambient, insulation, pump flow... yadda yadda...!!) That would take me about 40 hours to heat up. 

So I think my calculations are right, it would be 2kW x 40 hours x £0.33 - which gives me just over £26 to heat it. 😭

I want to use my hottub over the Christmas period, but the costs are scaring me still! 🥺

**Reply by wyliecoyoteuk** [reply]
I am on an EV specific tariff (octopus Go), daytime price is 38p/KWh, but between 00:30 and 4:30 AM it is 7.5p/KWh.
My wife has limited mobility and needs the tub for exercise.
Which is why this hack is so important to me, it allows me to heat the tub to 40°C in that window, and even in the current weather of 0°C or lower at night, the tub holds that for most of the day. It is heavily insulated and inside a summer house.

**Comment by heijjan** [comment]
We’ve had Helsinki spa over the winter in Finland without any problems. Coldest day has been -29C so far. Our tub is idling at around 20-24 degrees depending on the electricity price and is heated only during the cheap hours. About 12 hours prior to having a bath I set it for constant heating. 

At our rates (spot price) and this automation average cost for the spa has been around 1 euro/day.
